### PR TITLE
Update building-ardupilot-onwindows10.rst

### PR DESCRIPTION
**Pull request by auturgy**
removed Make and APM, clarified some things (like WSL Ubuntu Bash not being able to access USB).

### CODE REVIEW

File: dev/source/docs/building-ardupilot-onwindows10.rst [hunk 1]
```diff
@@ -1,29 +1,24 @@
 
-.. _building-ardupilot-onwindows10:
+.. _building-ardupilot-on-windows10:
```
Comment (OXINARF) [inline dev/source/docs/building-ardupilot-onwindows10.rst]: You can't just change the reference name without changing it where it is linked.

File: dev/source/docs/building-ardupilot-onwindows10.rst [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
 
 =======================================
-Building ArduPilot on Windows with Bash
+Building ArduPilot on Windows with waf and Bash
 =======================================
```
Comment (khancyr) [inline dev/source/docs/building-ardupilot-onwindows10.rst]: you need to expend === to match "Building ArduPilot on Windows with waf and Bash" length

**Comment by auturgy** [comment]
Won't hurt to put them back in.

Regards,

James

**Comment by khancyr** [comment]
LGTM ! thanks

**Comment by OXINARF** [comment]
I squashed, changed to commit message to something nicer and merged, thanks!

**Pull request closed without merging**